### PR TITLE
(+) blog page

### PR DESCRIPTION
**Pull request by fransachmadhw**
halaman blog doang. tapi belum responsive

### CODE REVIEW

File: src/components/HeroBlog.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          {/* Bingung nganu search bar nya hahaha */}
+          {/* <div className="mt-[50px] flex flex-col gap-3">
+            <h3 className="text-white text-lg">Cari artikel di sini</h3>
+          </div> */}
+        </div>
+        <div className="w-1/3">
+          <div className="w-full h-[300px] relative">
+            <Image
+              alt="hero"
+              src={HeroImage}
```
Comment (vzrenggamani) [inline src/components/HeroBlog.jsx:26]: Should be fetched from image frontmatter?

File: src/components/Footer.jsx
Comment (vzrenggamani) [inline src/components/Footer.jsx:1]: The link should be populated from an array of links like the `navbar` components

File: src/pages/blog/index.jsx
Comment (vzrenggamani) [inline src/pages/blog/index.jsx:1]: We should pull the `excerpt` and `date` from the frontmatter?

File: src/styles/globals.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
 @tailwind components;
 @tailwind utilities;
 
-* {
-  outline: 1px solid red;
-  background-color: rgba(100, 0, 0, 0.1);
+body {
+  outline: 0px solid red;
+  background-color: #021A28;
```
Comment (vzrenggamani) [inline src/styles/globals.css:7]: Background color should be iniatlized from `_documents.js`?

File: src/components/HeroBlog.jsx (hunk 1 above)
Comment (fransachmadhw) [reply src/components/HeroBlog.jsx:26]: Just a static hero. Please see the design first

File: src/pages/blog/index.jsx
Comment (fransachmadhw) [reply src/pages/blog/index.jsx:1]: Nanti kan bisa bang

**Review by vzrenggamani — APPROVED** [review]
lgtm! 😃 

Tracked feat-req on #19

**Pull request merged**